resources/nl/cemu.po

```diff
@@ -984,929 +1181,1278 @@ msgstr ""
 "bestaat dat je verbannen kan worden. Ga alleen door als je wilt riskeren om "
 "je online toegang kwijt te raken met je Wii U en/of je NNID."
 
-#: GeneralSettings2.cpp:520
+#: GeneralSettings2.cpp
 msgid "Account settings"
 msgstr "Account instellingen"
 
-#: GeneralSettings2.cpp:527
+#: GeneralSettings2.cpp
 msgid "Active account"
 msgstr "Actief account"
 
-#: GeneralSettings2.cpp:539
+#: GeneralSettings2.cpp
+msgid "Create"
+msgstr "Aanmaken"
+
+#: GeneralSettings2.cpp InputSettings.cpp
+msgid "Delete"
+msgstr "Verwijder"
+
+#: GeneralSettings2.cpp
+msgid "Enable online mode"
+msgstr "Schakel online modus in"
+
+#: GeneralSettings2.cpp
 msgid "Account information"
 msgstr "Account informatie"
 
-#: GeneralSettings2.cpp:548
-msgid "Id"
-msgstr "Id"
+#: GeneralSettings2.cpp
+msgid "The persistent id is the internal folder name used for your saves"
+msgstr ""
+"De persistente id is de interne map naam die wordt gebruikt voor je saves"
+
+#: GeneralSettings2.cpp
+msgid "Mii name"
+msgstr "Mii naam"
 
-#: GeneralSettings2.cpp:551
-msgid "E-Mail"
-msgstr "E-mailadres"
+#: GeneralSettings2.cpp
+msgid "The mii name is the profile name"
+msgstr "De mii naam is je profiel naam"
```

Comment:
I think that 'mii' should start with an uppercase letter in every language, regardless of how the original string looks like

Reply:
I personally disagree. The general name of the product might be Mii (which I do think should be capitalized too), but you're referring to an individual "species" or type of an object.

Reply:
Just trying to have a discussion here. I don't think capitalization really matters, so I'll change it to Mii like you've suggested.😄

Reply:
The goal here should be to be consistent with how the term is used "in the wild", and since I couldn't find any place where Mii is not capitalized (I quickly searched through [Nintendo Dutch website](https://www.nintendo.nl/Zoeken/Zoeken-299117.html?q=mii&f=147397-3958)), I think that's enough reason to capitalize it (and it's not even a common noun).